backend/src/services/token/common/types/data.type.ts

```diff
@@ -0,0 +1,5 @@
+type Data = {
```

Comment:
```suggestion
type TokenPayload = {
```
and move it to the `shared/src/common/type/token`